katalog/nginx/bases/configs/rbac/role.yml

```diff
@@ -55,21 +55,6 @@ rules:
       - get
       - list
       - watch
-  - apiGroups:
-      - ""
-    resourceNames:
-      - ingress-controller-leader
-    resources:
-      - configmaps
-    verbs:
-      - get
-      - update
-  - apiGroups:
-      - ""
-    resources:
-      - configmaps
-    verbs:
```

Comment:
There are no difference from rendered helm chart v4.11.2 and v4.11.3 but the following rule has been removed. I don't know if it was kept in past to retrocompatibility. @nutellinoit @ralgozino @sbruzzese902